### PR TITLE
Automated update by SDK Generator - daccf38

### PR DESCRIPTION
**Pull request by elsmr**

### CODE REVIEW

File: src/gen/docs/apis/VaultApi.md [hunk 1]
```diff
@@ -748,27 +748,27 @@ const apideck = new Apideck({
 
 const params = {
   session: {
```
Comment (Gdewilde) [inline src/gen/docs/apis/VaultApi.md:750]: shouldn't this be sessionRequest?

**Review by Gdewilde — CHANGES_REQUESTED** [review]

File: src/gen/docs/apis/VaultApi.md (hunk 1 above)
Comment (elsmr) [reply src/gen/docs/apis/VaultApi.md:750]: yeah, looks like the "root" parameter key for request bodies is wrong for many of these examples. They come from `x-typescript-params` in unify, I'll log an issue

Comment (elsmr) [reply src/gen/docs/apis/VaultApi.md:750]: going to merge this one, already makes the docs more correct than before

Comment (Gdewilde) [reply src/gen/docs/apis/VaultApi.md:750]: ok

**Pull request merged**